third_party/k8s.io/apimachinery/pkg/api/resource/quantity.go

```diff
@@ -25,6 +25,8 @@ import (
 	"strconv"
 	"strings"
 
+	cbor "k8s.io/apimachinery/pkg/runtime/serializer/cbor/direct"
```

Comment:
I thought that would have introduced a new dependency, but that's not true.

I checked-out the PR, run `go mod tidy` and then inspected the `go.sum` but I didn't find any reference to [`github.com/fxamacker/cbor/v2`](https://github.com/kubernetes/apimachinery/blob/a8f449e276fe566efddb149992049c78f0088492/pkg/runtime/serializer/cbor/cbor.go#L35C2-L35C32)